code/style.css

```diff
@@ -1,28 +1,28 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: #FF9E00;
+  --secondary: #fad08d; 
 }
 
 body {
   display: flex;
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
+  background: #f0daf1;
 }
 
 h1 {
   font-size: 42px;
   font-weight: 500;
   line-height: 48px;
   margin: 10px 0;
-  color: white;
+  color: #63458A;
```

Comment:
You can use global variables as in the beginning of the code for a more uniform way of styling.

Reply:
Very good point, I will remember that for the future